### PR TITLE
Isb cgc prod elee

### PR DESCRIPTION
**Pull request by eleeisb**

### CODE REVIEW

File: isb_cgc/urls.py [hunk 1]
```diff
@@ -19,16 +19,14 @@
...
-from django.views.generic.base import TemplateView
 
 from . import views
 
 admin.autodiscover()
 
 urlpatterns = [
 
-    url(r'^$', views.landing_page, name='landing_page'),
-    url(r'robots.txt', TemplateView.as_view(template_name="robots.txt", content_type="text/plain"), name='robots'),
```
Comment (s-paquette) [inline isb_cgc/urls.py:31]: Did we add the robots.txt in elsewhere?

File: static/js/cohort_barcodes_details.js [hunk 2]
```diff
@@ -19,21 +19,23 @@
...
+        // bootstrap: 'libs/bootstrap.min',
+        // jqueryui: 'libs/jquery-ui.min',
+        // session_security: 'session_security/script',
+        // underscore: 'libs/underscore-min',
         ajv: 'libs/ajv.bundle',
-        base: 'base',
-        dataTables:'libs/jquery.dataTables.min'
+        // base: 'base',
+        // dataTables:'libs/jquery.dataTables.min'
+        'datatables.net': ['//cdn.datatables.net/1.10.19/js/jquery.dataTables.min', 'libs/jquery.dataTables.min'],
```
Comment (s-paquette) [inline static/js/cohort_barcodes_details.js:30]: Found a sneaky CDN

File: isb_cgc/urls.py (hunk 1 above)
Comment (eleeisb) [reply isb_cgc/urls.py:31]: robots.txt along with the url code are added back.

File: static/js/cohort_barcodes_details.js (hunk 2 above)
Comment (eleeisb) [reply static/js/cohort_barcodes_details.js]: removed cdn library

**Pull request merged**